packages/react/src/Image/Image.tsx

```diff
@@ -6,15 +6,25 @@
 import clsx from 'clsx'
 import { forwardRef } from 'react'
 import type { ForwardedRef, ImgHTMLAttributes } from 'react'
+import { AspectRatioProps } from '../AspectRatio'
 
-export type ImageProps = {
+export type ImageProps = AspectRatioProps & {
```

Comment:
Hmm… an image should always have an aspect ratio configured, right? It’s not like it’s okay to ‘forget’ choosing one and using an image of arbitrary size – then the entire idea of having a fixed set of allowed ratios is gone.

Let’s make 16/9 the default.

Reply:
Does everyone agree, @alimpens @dlnr?

This will make that `object-fit: cover` is also always applied as per [this comment](https://github.com/Amsterdam/design-system/pull/1593#discussion_r1813504108).

Reply:
I think I agree. It has the consequence that images will be distorted if they do not have one of our allowed ratio's. But maybe that's preferable to not setting a ratio at all.